keyboards/dozen0/rules.mk

```diff
@@ -8,26 +8,25 @@ MCU = atmega32u4
 #   LUFA DFU     lufa-dfu
 #   QMK DFU      qmk-dfu
 #   ATmega32A    bootloadHID
-#   ATmega328P   USBasp
-BOOTLOADER = atmel-dfu
+BOOTLOADER = caterina
```

Comment:
Lines 10-11 should be changed to:

```Makefile
#   ATmega32A    bootloadHID
#   ATmega328P   USBasp
BOOTLOADER = caterina
```

There was a line addition here on the previous review that didn't get added.

Reply:
I'm sorry, I overlooked that.